zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/Collection.java

```diff
@@ -441,10 +443,8 @@ public Release populateRelease(CollectionReader reader, CollectionWriter collect
                 InputStream dataStream = resource.getData()
         ) {
             Release release = (Release) ContentUtil.deserialiseContent(dataStream);
-            logInfo("Release identified for collection")
-                    .collectionId(this)
-                    .addParameter("title", release.getDescription().getTitle())
-                    .log();
+            info().data("collectionId", this).data("title", release.getDescription().getTitle())
```

Comment:
I think `.data("collectionId", this)....` should be `.data("collectionId", this.getDescription().getId())....` otherwise we will have the entire collection in the json which I don't we want/need.